src/apps/kfone-customer-support/.env.example

```diff
@@ -7,4 +7,8 @@ REACT_APP_ASGARDEO_CALLBACK_URL=<CALLBACK URL AFTER A SUCCESSFUL AUTHENTICATION>
 REACT_APP_CHOREO_CLIENT_ID=<CONSUMER ID TAKEN FROM ASGARDEO CONSOLE>
 REACT_APP_BASE_API_ENDPOINT=<BASE URL OF CHOREO HOSTED API>
 REACT_APP_CHOREO_ORGANIZATION=<ORGANIZATION OF CHOREO HOSTED API>
-REACT_APP_CHOREO_TOKEN_ENDPOINT=<CHOREO TOKEN ENDPOINT>
+
+# STS Token Endpoint is the endpoint to obtain security tokens from a security token service.
+# You can obtain this value from your identity provider.
+# Ex: For Choreo, https://sts.choreo.dev/oauth2/token
+REACT_APP_STS_TOKEN_ENDPOINT='<your_sts_token_endpoint_here>'
```

Comment:
Are quotes needed here? And better to have the description similar to the other configs.

Reply:
Planning to change the description of the other configs as well.
Quotes are not really needed but seem to give better highlighting.
Will try to remove it.

Reply:
Fixed with https://github.com/asgardeo-samples/asgardeo-enterprise-ciam-sample/pull/13/commits/452eaec7b871daf10e5910933d58e0bb77fa297c